everyBot/cogs/dnd/dnd.py

```diff
@@ -54,6 +56,30 @@ async def rollto(self, ctx, *, text):
 
         return await ctx.send(embed=embed)
 
+    """ Roll """
+    @commands.command(
+        usage="[dice]",
+        description="Roll a specified dice (eg. 2d6)"
+    )
+    @commands.check(check_disabled)
+    async def roll(self, ctx, *, text):

```

Comment:
`*, text` isn't the right way to grab the dice here, since the dice can only be 1 word you can just have the parameters as follows:
`roll(self, ctx, dice)`